src/resources/vue/components/ItemCard.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="item-card">
+    <div class="left">
+      <Icon icon="book" size="90" color="white" />
+      <div class="info">
+          <p>{{ title }}</p>
+          <p>{{ description }}</p>
+      </div>
+    </div>
+    <div v-if="warning === 'true'" class="right">
```

Comment:
Als warning een boolean is, waarom doe je dan `=== 'true'`?

Reply:
Dit was met chris besloten, je kan geen booleans meegeven in props

Reply:
gebruik `:` :)